### PR TITLE
Plug in LogSource and TraceSource into JTS and formatting done in handlers

### PR DESCRIPTION
**Pull request by Yushan-Lin**

### CODE REVIEW

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_sXPb4A0TEeiTR7j3BTw-RQ&tab=com.ibm.team.build.web.ui.internal.editors.result.test.TestProvider

File: dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java [hunk 1]
```diff
@@ -407,6 +489,63 @@ public String messageLogFormat(LogRecord logRecord, String formattedVerboseMsg)
...
+                    loggerName = kvp.getValue();
+                } else if (kvp.getKey().equals(LogFieldConstants.IBM_CLASSNAME)) {
+                    srcClassName = kvp.getValue();
+                } else if (kvp.getKey().equals(LogFieldConstants.THROWABLE)) {
+                    throwable = kvp.getValue();
+                }
+
+            }
+        }
+        name = nonNullString(srcClassName, loggerName);
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java]: this should be nonNullString(loggerName, srcClassName) to match method above

File: dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java [hunk 2]
```diff
@@ -736,13 +1070,100 @@ private String getMarker(LogRecord logRecord) {
...
         }
     }
+
+    /**
+     * Outputs filteredStream of genData
+     *
+     * @param genData object to filter
+     * @return filtered message of the genData
+     */
+    protected String filteredStreamOutput(GenericData genData) {
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java]: should this be renamed to formatStreamOutput?

File: dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java [hunk 3]
```diff
@@ -736,13 +1070,100 @@ private String getMarker(LogRecord logRecord) {
...
+
+            if (p instanceof KeyValuePair) {
+
+                kvp = (KeyValuePair) p;
+                if (kvp.getKey().equals("message")) {
+                    txt = kvp.getValue();
+                } else if (kvp.getKey().equals("loglevel")) {
+                    loglevel = kvp.getValue();
+                } else if (kvp.getKey().equals("throwable")) {
+                    loglevel = kvp.getValue();
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java]: this line seems odd... just want to double check to make sure you meant to set loglevel to the value of throwable

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_PKlGcBAeEeiStsw2He7iFw

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_YgEPIBAeEeiStsw2He7iFw

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_v-87cBAeEeiStsw2He7iFw

**Comment by Yushan-Lin** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_BObeYBBPEeiStsw2He7iFw
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_80V4sBBOEeiStsw2He7iFw

**Pull request closed without merging**